packages/suite-desktop-core/src/libs/connect-ws.ts

```diff
@@ -229,6 +231,7 @@ export const exposeConnectWs = ({
 
                 // wait for response
                 const response = await messages[message.id].promise;
+                connectionPendingMessages.delete(message.id);
```

Comment:
not sure whether there is something above this line that can throw which could lead to desync in state of connectionPendingMessages